site/docs/src/json/entity-types/response.json

```diff
@@ -0,0 +1,31 @@
+{
+  "entityType": {
+    "data": {
+      "createdBy": "jared@fusionauth.io",
+    },
+    "id": "8174f72f-5ecd-4eae-8de8-7fef597b3473",
+    "insertInstant": 1595361142909,
+    "jwtConfiguration": {
+      "accessTokenKeyId": "a7516c7c-6234-4021-b0b4-8870c807aeb2",
+      "enabled": true,
+      "timeToLiveInSeconds": 3600
+    },
+    "lastUpdateInstant": 1595361143101,
+    "name": "Customer",
+    "permissions": [
+      {
+        "entityTypeId": "8174f72f-5ecd-4eae-8de8-7fef597b3473",
+        "id": "ce485a91-906f-4615-af75-81d37dc71e90",
+        "insertInstant": 1595361142909,
+        "isDefault": false,
+        "lastUpdateInstant": 1595361143101,
+        "name": "admin"
```

Comment:
`entityTypeId` will not be in the permission json.  Here is an example with all fields:

```
{
  "id": "ce485a91-906f-4615-af75-81d37dc71e90",
  "name": "admin",
  "description": "This permission grants admin privileges",
  "isDefault": false,
  "insertInstant": 1595361142909,      
  "lastUpdateInstant": 1595361143101
}
```

Reply:
Hiya @bguyza 

Ah, my bad: https://github.com/FusionAuth/fusionauth-java-client/blob/master/src/main/java/io/fusionauth/domain/EntityTypePermission.java looks like it lists everything.